### PR TITLE
Add some papers from ICSE 2024 and FSE 2024

### PR DESCRIPTION
**Pull request by jeffchen006**
I added some papers from FSE 2024 and ICSE 2024, please review. Thanks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -547,6 +547,19 @@ Carsten Binnig, Arvind Arasu, Donald Kossmann, Ravi Ramamurthy. VLDB'19.
...
+
+[Smart Contract] [Are We There Yet? Unraveling the State-of-the-Art Smart Contract Fuzzers](https://arxiv.org/pdf/2402.02973.pdf). Shuohan Wu, Zihao Li, Luyi Yan, Weimin Chen, Muhui Jiang, Chenxu Wang, Xiapu Luo, Hao Zhou. ICSE '2024
+
+[Smart Contract] [Smart Contract and DeFi Security Tools: Do They Meet the Needs of Practitioners?](https://arxiv.org/pdf/2304.02981). Stefanos Chaliasos, Marcos Antonios Charalambous, Liyi Zhou, Rafaila Galanopoulou, Arthur Gervais, Dimitris Mitropoulos, Ben Livshits. ICSE '2024
+
+[Smart Contract] [Safeguarding DeFi Smart Contracts against Oracle Deviations](https://arxiv.org/pdf/2401.06044). Xun Deng, Sidi Mohamed Beillahi, Cyrus Minwalla, Han Du, Andreas Veneris, Fan Long. ICSE '2024
+
+[Smart Contract] [GPTScan: Detecting Logic Vulnerabilities in Smart Contracts by Combining GPT with Program Analysis](https://arxiv.org/pdf/2308.03314). Yuqiang Sun, Daoyuan Wu, Yue Xue, Han Liu, Haijun Wang, Zhengzi Xu, Xiaofei Xie, Yang Liu. ICSE '2024
+
+
```
Comment (gitferry) [inline README.md:562]: empty line

File: README.md [hunk 2]
```diff
@@ -561,6 +574,15 @@ Carsten Binnig, Arvind Arasu, Donald Kossmann, Ravi Ramamurthy. VLDB'19.
...
 
+[Smart Contract] [Static Application Security Testing (SAST) Tools for Smart Contracts: How Far Are We?)](https://arxiv.org/pdf/2404.18186). Kaixuan Li, Yue Xue, Sen Chen, Han Liu, Kairan Sun, Ming Hu, Haijun Wang, Yang Liu, Yixiang Chen. ESEC/FSE '24.
+
+[Smart Contract] [Demystifying Invariant Effectiveness for Securing Smart Contracts](https://arxiv.org/abs/2404.14580). Zhiyang Chen, Ye Liu, Sidi Mohamed Beillahi, Yi Li, Fan Long. ESEC/FSE '24.
+
+[Smart Contract] [Efficiently Detecting Reentrancy Vulnerabilities in Complex Smart Contracts](https://arxiv.org/abs/2403.11254). Zexu Wang, Jiachi Chen, Yanlin Wang, Yu Zhang, Weizhe Zhang, Zibin Zheng. ESEC/FSE '24.
+
+
+
+
```
Comment (gitferry) [inline README.md:585]: empty lines

**Review by gitferry — CHANGES_REQUESTED** [review]
Nice addtion! Please address the comments:

**Comment by jeffchen006** [comment]
Empty lines fixed

**Review by gitferry — APPROVED** [review]
Nice additon!

**Pull request merged**